etl/PAIMS_PIMS/PAIMS_PIMS/Project.params

```diff
@@ -79,7 +79,7 @@
       <SSIS:Property
         SSIS:Name="Sensitive">0</SSIS:Property>
       <SSIS:Property
-        SSIS:Name="Value">_YOUR_USER_HERE_</SSIS:Property>
+        SSIS:Name="Value">desmith</SSIS:Property>
```

Comment:
This is where project params (as reflected by the file name) are presented in the UI:
![image](https://user-images.githubusercontent.com/41091511/140543431-58f1537f-5393-4823-951a-3e87e82c7b0b.png)

You should also be able to delete unused parameters from this screen.
@fieranmason

Reply:
@fieranmason this hasn't been resolved yet. Please resolve so that we can merge.